src/components/Feed.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useRef, useState } from "react";
+import { useEffect } from "react";
+
+import styled from "styled-components";
+
+const Feed = (props) => {
+  const [commentText, setCommentText] = useState("");
+  const [commentList, setCommentList] = useState(props.comments);
+  const [isLoad, setIsLoad] = useState(true);
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    onLoadHandler();
+  }, []);
+
+  const onLoadHandler = () => {
+    const loadImage = new Image();
+    loadImage.src = props.img;
+    loadImage.onload = () => {
+      setIsLoad(false);
+    };
+  };
+
+  const commentInputChangeHandler = () => {
+    setCommentText(inputRef.current.value);
+  };
+
+  const findLastItem = () => {
+    const lastItem = commentList[commentList.length - 1];
+    if (lastItem !== undefined) {
+      return lastItem.id + 1;
+    } else {
+      return 1;
+    }
+  };
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+    setCommentList([
+      ...commentList,
+      { id: findLastItem(), userName: "superUser", comment: commentText },
+    ]);
+    setCommentText("");
+  };
+
+  if (isLoad) return <></>;
+
+  return (
+    <Container>
+      <FeedHead>
+        <ProfileWrap>
+          <ProfileCircle></ProfileCircle>
+          <span>{props.userName}</span>
+        </ProfileWrap>
+        <div>
+          <button></button>
+        </div>
+      </FeedHead>
+      <FeedImage>
+        <img alt="이미지" src={props.img} />
+      </FeedImage>
+      <FeedComment>
+        <CommentList>
+          <CommentUl>
+            {commentList.map((item) => (
+              <CommentLi key={item.id}>
+                <UserName>{item.userName}</UserName>
+                <UserComment>{item.comment}</UserComment>
+              </CommentLi>
+            ))}
+          </CommentUl>
+        </CommentList>
+        <form onSubmit={submitHandler}>
+          <AddComment>
+            <CommentInput
+              ref={inputRef}
+              onChange={commentInputChangeHandler}
```

Comment:
요런 건 취향차이일 수 있지만 저는 아래처럼 하는 게 보기가 편하더라구요!
```suggestion
              onChange={setCommentText(inputRef.current.value)}
```

Reply:
클린코드에서 추구하는 방향과 조금 다른 것 같아용